cvat/schema.yml

```diff
@@ -7140,6 +7866,66 @@ components:
             type: string
             minLength: 1
           writeOnly: true
+    PatchedQualitySettingsRequest:
+      type: object
+      properties:
+        iou_threshold:
+          type: number
+          format: double
+          description: Used for distinction between matched / unmatched shapes
+        oks_sigma:
+          type: number
+          format: double
+          description: |
+            Like IoU threshold, but for points.
+            The percent of the bbox area, used as the radius of the circle around the GT point,
+            where the checked point is expected to be.
+            Read more: https://cocodataset.org/#keypoints-eval
+        line_thickness:
+          type: number
+          format: double
+          description: |
+            Thickness of polylines, relatively to the (image area) ^ 0.5.
+            The distance to the boundary around the GT line,
+            inside of which the checked line points should be
+        low_overlap_threshold:
+          type: number
+          format: double
+          description: Used for distinction between strong / weak (low_overlap) matches
+        oriented_lines:
```

Comment:
```suggestion
        orient_lines:
```

Reply:
The parameter indicates that the line direction is important in the task. The suggested name would mean that lines are being changed somehow.

Reply:
Sounds good.